web_poet/pages.py

```diff
@@ -7,8 +7,15 @@
 
 
 class Injectable(abc.ABC):
-    """Injectable objects are automatically built and passed as arguments to
-    callbacks that requires them.
+    """
```

Comment:
We're usually starting the docstring with the text rather than with a blank line.

I think we should try to commit to one style.